src/components/Title.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import App from '../containers/App';
+
+class Title extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: "'TO DO' APPLICATION"
+        };
+    }
+
+    render() {
+        const app = new App(),
```

Comment:
App jest tworzone tylko raz - wewnątrz index.js - tutaj nie powinieneś go tworzyć. To co tutaj zrobiłeś to tzw "circular dependency" czyli App potrzebuje Title a Title potrzebuje App, nie jest to dobra praktyka.

Wewnątrz tego komponentu wystarczy wyświetlić tytuł TODO listy.

Jeśli chcesz wyświetlić elementy listy to powinieneś to zrobić wewnątrz App nie tutaj